.buildkite/pipeline.yml

```
@@ -31,6 +31,15 @@ steps:
     env:
       BUILDKITE_SPLITTER_TEST_FILE_PATTERN: "spec/acceptance/**/*_spec.rb"
 
+  - label: "Everything Else"
+    command: .buildkite/steps/run
+    key: everything_else
+    agents:
+      queue: "hosted-queue"
+    parallelism: 2
+    env:
+      BUILDKITE_SPLITTER_TEST_FILE_EXCLUDE_PATTERN: "{spec/acceptance/**/*_spec.rb,spec/unit/**/*_spec.rb}"
```

alternatively, you can do this:
```suggestion
      BUILDKITE_SPLITTER_TEST_FILE_EXCLUDE_PATTERN: "spec/{acceptance,unit}/**/*_spec.rb"
```

---

oh yeah, that's much nicer